test/e2e/admission.go

```diff
@@ -51,4 +54,74 @@ var _ = Describe("Job E2E Test: Test Admission service", func() {
 			"Job queue attribute would default to 'default' ")
 	})
 
+	It("Create default-scheduler pod", func() {
```

Comment:
what is this case for?

Reply:
test normal pod will be created succeeded for AdmitPod